Tests/DependencyInjection/SonataAdminExtensionTest.php

```diff
@@ -16,6 +16,20 @@
 
 class SonataAdminExtensionTest extends AbstractExtensionTestCase
 {
+    public function testHasServiceDefinitionForLockExtension()
+    {
+        $this->container->setParameter('kernel.bundles', array());
+        $this->load(array('options' => array('lock_protection' => true)));
+        $this->assertContainerBuilderHasService('sonata.admin.lock.extension');
```

Comment:
seems like this method is not defined under our composer.json minimum requirements.

try to update it on composer.json to: ´´"matthiasnoback/symfony-dependency-injection-test": "^0.7.6 || ^1.1",``

Is what @greg0ire did on his PR to work with simple-phpunit